### ISSUE TITLE
[Suggestion] Instead of sharing the whole monitor, you can select what window Parsec Soda will show to users

### ISSUE THREAD

**Issue by MatadorProBr** [FlavioFS/ParsecSoda#29]
Why you have to use your whole monitor just to share a game instead of just choosing the game window?

Most people want to share only the game window because sometimes they want to change the window to see for example: Discord and if you change the window, all of the users that are connected on your room will see all actions that you make on that specific app and I don't think this is good if you want privacy so why don't put a option (optionally) that you can just tell to Parsec Soda: "only share that specific window ok?" like: OBS and if this can't be possible I want to know why, since I'm a bit curious.

Maybe someone already give this suggestion but I searched on GitHub, Discord and I found nothing so sorry if this suggestion is a duplicate of a already suggestion.

**Comment by borjita2019** [comment]
i think there's no feature requests that are the same as this.. so i think you are the first one!

**Comment by unuruu5760** [comment]
Even in the original parsec it's imperfect and unstable. So the melon soda was made to show the full monitor screen.
If you only want to show the game window to your guests, you can use obs to display it on a second monitor (virtual monitor).

**Comment by MatadorProBr** [comment]
> you can use obs to display it on a second monitor (virtual monitor).

Yeah but I think this is not the best solution and it can harm your performance of the game

**Comment by unuruu5760** [comment]
> Yeah but I think this is not the best solution and it can harm your performance of the game

Currently that is the best solution.
The function you are requesting works unreliably even with original parsec. In particular, it only works with OpenGL. So he probably won't be able to complete the function you are asking for.
